srcpkgs/power-profiles-daemon/template

```diff
@@ -14,7 +14,7 @@ license="GPL-3.0-or-later"
 homepage="https://gitlab.freedesktop.org/hadess/power-profiles-daemon"
```

Comment:
This should be updated to `https://gitlab.freedesktop.org/upower/power-profiles-daemon`